Disponibilizada la documentacion de Wix para hispanos y fue actualiza… (#1217)
mercadopago/devsite-docs: guides/resources/changelog/current-year.en.md

## July 2021

## July 15th

> ----[mla,mlc,mco,mpe,mlu]----

Reviewer: ```suggestion
----[mla, mlc, mco, mpe, mlu]----
```

>  CHANGELOG

Reviewer: ```suggestion
> CHANGELOG
```

> 
> Wix as a new payment method in Checkout Pro
>
> NEWS: NEWS
>

> > PRODUCT: WIX

Reviewer: ```suggestion
> PRODUCT: WIX

```

Now you can integrate Mercado Pago into your Wix store. With it, you can offer your customers the possibility to pay directly on the Mercado Pago site.

[Go to Wix documentation](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/plugins/unofficial/wix)

------------

----[mlb]----
> CHANGELOG
> 
> New Wix documentation
>

> > NEWS: NEWS

Reviewer: ```suggestion
> NEW_VERSION: NEW VERSION
```

>

> > PRODUCT: WIX

Reviewer: ```suggestion
> PRODUCT: WIX

```

## July 2021

## July 15th
----[mla,mlc,mco,mpe,mlu]----
> CHANGELOG
> 
> Wix as a new payment method in Checkout Pro
>
> NEWS: NEWS
>
> PRODUCT: WIX

> Now you can integrate Mercado Pago into your Wix store. With it, you can offer your customers the possibility to pay directly on the Mercado Pago site.

Reviewer 2: ```suggestion
Now you can integrate Mercado Pago into your Wix store. With this virtual platform, you can offer your customers the possibility to pay directly on the Mercado Pago site.
```
